src/main/java/net/kodehawa/mantarobot/commands/custom/v3/ast/NodeVisitor.java

```diff
@@ -17,6 +17,7 @@
 
 package net.kodehawa.mantarobot.commands.custom.v3.ast;
 
+@SuppressWarnings("unused")
```

Comment:
@natanbc do we need this class? Doesn't seem to be used

Reply:
This interface is used. The one we talked about is SerializerVisitor